index.html

```diff
@@ -9,23 +9,37 @@
   <title>Meet landing page</title>
 </head>
 <body>
-  Meet makes it easy to connect with others face-to-face virtually and collaborate across any device.
-
-  Download v1.3
-  What is it?
-
-  01
-  Built for modern use
-
-  Smarter meetings, all in one place
-
-  Send messages, share files, show your screen, and record your meetings — all in one 
-  workspace. Control who can join with invite-only team access, data encryption, and data export.
-
-  02
-  Experience more together
-  Stay connected with reliable HD meetings and unlimited one-on-one and group video sessions.
-
-  Download v1.3
+  <main class="container">
```

Comment:
This page should look like this
![Screenshot from 2021-09-21 21-16-19](https://user-images.githubusercontent.com/90243506/134234033-6d252991-b0dc-4313-82e4-cd75db157215.png)
: